src/main/kotlin/com/sourcegraph/cody/config/CodyWindowManager.kt

```diff
@@ -0,0 +1,30 @@
+package com.sourcegraph.cody.config
+
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.project.Project
+import com.sourcegraph.cody.agent.CodyAgent
+import com.sourcegraph.cody.agent.CodyAgentService
+import com.sourcegraph.cody.agent.protocol_generated.Window_DidChangeFocusParams
+import java.awt.event.WindowAdapter
+import java.awt.event.WindowEvent
+
+class CodyWindowAdapter(private val project: Project) : WindowAdapter() {
```

Comment:
Can you also rename file to match class name?

Reply:
done